apps/api/src/app/inbox/usecases/session/session.usecase.ts

```diff
@@ -148,6 +184,19 @@ export class Session {
     };
   }
 
+  private async getApplicationIdentifier(command: SessionCommand): Promise<string> {
+    const isKeylessInitialize = !command.applicationIdentifier;
+    const isKeyless = command.applicationIdentifier?.includes(this.KEYLESS_ENVIRONMENT_PREFIX);
+    const isKeylessExpired = isKeyless ? await this.isKeylessExpired(command.applicationIdentifier) : false;
```

Comment:
```suggestion
    const isKeylessInitialize = !command.applicationIdentifier;
    if (!isKeylessInitialize) return command.applicationIdentifier;
```

Maybe we can return early and simplify the ternaries below here.

Reply:
this will not work as you expect it to work. 
there could be a state that will be `not isKeylessInitialize` but a `isKeyless` environment, in this case we still will want to validate if the keyless environment is expired or not.